### ISSUE TITLE
get the C function name in logs

### ISSUE THREAD

**Issue by gdesmott** [standard-ai/tracing-gstreamer#4]
Would it be possible to get the C function name in logs as we do with the standard logs? I find the logs quite hard to follow without those.

**Comment by nagisa** [comment]
We populate the [`module` field](https://docs.rs/tracing/0.1.29/tracing/struct.Metadata.html#method.module_path) with the C function name for the events. I'm not sure if there's a good way to output this data with the `tracing_subscriber::fmt`. Your best bet is to ask on the [tracing discord](https://discord.gg/cZ9GeY9T), I think.

**Comment by gdesmott** [comment]
Thanks, I [opened a ticket](https://github.com/tokio-rs/tracing/issues/1628), let's see.
